modules/javafx.base/src/main/java/javafx/beans/property/BooleanProperty.java

```diff
@@ -85,14 +85,16 @@ public void setValue(Boolean v) {
      */
     @Override
     public void bindBidirectional(Property<Boolean> other) {
+        Objects.requireNonNull(other, "Property cannot be null");
```

Comment:
While the same check also happens in `BidirectionalBinding`, this improves the error message (otherwise the message would be "Both properties must be specified").